modules/fscloud/variables.tf

```diff
@@ -195,3 +195,15 @@ variable "location" {
   description = "The region in which the network zone is scoped"
   default     = null
 }
+
+variable "kms" {
+  type        = list(string)
+  description = "List specific endpoint types for target services, valid values for endpoints are 'public', 'private' or 'direct'"
+  default     = ["kms", "hs-crypto"]
```

Comment:
I would default to hs-crypto only. Having both makes little sense in real world as well.

Reply:
ok resolved.